### PR TITLE
WRP-9583: Fixed MediaPlayer knob on focus for type "tiny" 

### PR DESCRIPTION
**Pull request by ion-andrusciac-lgp**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [X] Documentation was verified or is not changed
* [X] UI test was passed or is not needed
* [X] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Fixed MediaPlayer, when type = "tiny" slider knob overlays the text when knob is active and skin is Silicon

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
The knob was resized in order not to overlay the text

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
The same problem was found and resolved on the skin Carbon

### Links
[//]: # (Related issues, references)
WRP-9583

### Comments

Enact-DCO-1.0-Signed-off-by: Ion Andrusciac ion.andrusciac@lgepartner.com

### CODE REVIEW

File: styles/variables-base.less [hunk 1]
```diff
@@ -714,12 +714,17 @@
...
 @agate-slider-active-knob-transform: @agate-slider-knob-transform;
 @agate-slider-active-knob-width: @agate-slider-active-knob-height;
+@agate-slider-active-tiny-knob-height: @agate-slider-active-knob-height;
+@agate-slider-active-tiny-knob-width: @agate-slider-active-knob-width;
 @agate-slider-bg-transform: none;
 @agate-slider-border-radius: @agate-slider-height;
 @agate-slider-height: @agate-button-height;
 @agate-slider-horizontal-anchorpoint-height: 0px;  // This is used in `calc` which means it needs the unit specified to work correctly
 @agate-slider-horizontal-anchorpoint-width: 0px;  // This is used in `calc` which means it needs the unit specified to work correctly
+@agate-slider-horizontal-tiny-anchorpoint-height: @agate-slider-horizontal-anchorpoint-height;
```
Comment (daniel-stoian-lgp) [inline styles/variables-base.less:724]: this variable is not used. can you remove it?

File: styles/variables-silicon.less [hunk 2]
```diff
@@ -458,13 +458,19 @@
 @agate-slider-active-knob-border-radius: (@agate-slider-active-knob-height / 2);
 @agate-slider-active-knob-height: 60px;
 @agate-slider-active-knob-transform: @agate-translate-center scale(@agate-slider-knob-resting-state-scale);
+@agate-slider-active-tiny-knob-height: 21px;
+@agate-slider-active-tiny-knob-width: @agate-slider-active-tiny-knob-height;
 @agate-slider-border-radius: 0;
 @agate-slider-horizontal-anchorpoint-width: (@agate-slider-knob-height * @agate-slider-knob-resting-state-scale - 1px); // just a touch under the resting state scale, to avoid subpixel rendering overflow
+@agate-slider-horizontal-tiny-anchorpoint-width: (@agate-slider-tiny-knob-height * @agate-slider-knob-resting-state-scale - 1px);
+@agate-slider-horizontal-tiny-margin: 0 (@agate-slider-horizontal-tiny-anchorpoint-width / 2) 0 (@agate-slider-horizontal-tiny-anchorpoint-width / 2);
```
Comment (daniel-stoian-lgp) [inline styles/variables-silicon.less:466]: please use a 2- value margin :) since top/bottom and left/right are equal

**Comment by daniel-stoian-lgp** [comment]
please add a changelog entry. Thank you!

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by hoeunryuLG — APPROVED** [review]

**Pull request merged**